R/tm_front_page.R

```diff
@@ -53,11 +53,12 @@
 #'       footnotes = c("X" = "is the first footnote", "Y is the second footnote"),
 #'       show_metadata = TRUE
 #'     )
-#'   ),
-#'   header = tags$h1("Sample Application"),
-#'   footer = tags$p("Application footer"),
+#'   )
 #' )
 #'
+#' app <- modify_header(app, tags$h1("Sample Application"))
+#' app <- modify_footer(app, tags$p("Application footer"))
```

Comment:
We can use the `|>` in the documentation as related `teal` PR depends on 4.1 
```suggestion
#'   )
#' ) |>
#'   modify_header(tags$h1("Sample Application")) |>
#'   modify_footer(tags$p("Application footer"))
```

Reply:
👍🏽 also bumped the R version to `4.1` over here.